Gandalf/postinst

```diff
@@ -1,17 +1,28 @@
 #!/bin/bash
 
-#Declare variables
 
+#Variables
 RED='\033[0;31m' # Define RED as red
 NC='\033[0m' # Define NC as no color
-echo "Installing Gandalf..."
+
+#Main script
+echo "Installing the Gandalf application..."
 echo "Unpacking…"
+
 tar -xvf /var/mobile/Downloads/Gandalf102/Gandalf.app.tar.gz -C /Applications/
 echo "App installed..."
+
+
+echo "Cleaning up files.."
+
+rm -rf /var/mobile/Downloads/Gandalf102
+
+
 echo "Please Wait... running uicache..."
 uicache
+
+
 echo
 echo -e "${RED}--- Now please run 'gAndAlfh0LdC0mMand hold'! Just to be save. ---${NC}"
```

Comment:
Maybe make this with a variable too (like in the script for holding / installing gandalf)